### ISSUE TITLE
Voldemort does not build with last versions of java

### ISSUE THREAD

**Issue by AnastaciusWright** [voldemort/voldemort#500]
I was not able to build voldemort with Java 12, I had to switch to Java 8 to be able to do so. This information could be added to the QuickStart Page, until this bug is fixed.

**Comment by shreelakshmijoshi** [comment]
Hi @AlejandroFNadal !
I am a new open source contributor and would love to solve this issue, could you help me understand the issue better by providing some extra references ? 
Is this is issue open to contribute ?

**Comment by AlejandroFNadal** [comment]
Dear @shreelakshmijoshi , Thanks for your reply. I think this project is completely dead, considering that it has not had a single commit since 2017. I would recommend focusing your energies elsewhere, where they will be more useful.

**Comment by FelixGV** [comment]
That is correct, Voldemort is not actively developed anymore. It used to be primarily developed by LinkedIn (though there were a few external contributors here and there), and it got replaced by other systems within LinkedIn. Some of the Voldemort use cases migrated to [Pinot](https://github.com/apache/pinot), which is open-source and thriving, so you may want to check this out. Other use cases migrated to systems which are proprietary, including [Espresso](https://engineering.linkedin.com/blog/topic/espresso) (strongly consistent document store), [Galene](https://engineering.linkedin.com/blog/topic/galene) (search index), [Liquid](https://engineering.linkedin.com/blog/topic/knowledge_graph) (graph DB) and [Venice](https://engineering.linkedin.com/blog/topic/venice) (derived data store, serving all of the Voldemort Read-Only use cases and some others too). Some of these systems might become open-sourced in the future if there is demand for it.

**Comment by shreelakshmijoshi** [comment]
Hey @FelixGV ! I didn't know that, Thanks for the information !

**Comment by FelixGV** [comment]
That being said, if someone else wishes to take over the development of Voldemort, I don't think there would be much concern in sharing the access. Unless some of the old timers would prefer not to? @vinothchandar, @arunthirupathi, @afeinberg, etc...

**Comment by afeinberg** [comment]
Is the issue related to the use of unsafe in Voldemort read-only? I can take a stab at fixing it and putting that into my private fork.
